package.json

```diff
@@ -1,9 +1,6 @@
 {
   "name": "node-worker-pool",
-  "version": "2.4.5",
```

Comment:
I usually leave this for a separate commit so I can use `npm version` (which will both bump the version, create a notable commit message in commit history as a linchpin, and create a tag -- all for me)